examples/gateway/README.md

```diff
@@ -54,27 +54,27 @@ For full examples of configurations and supported endpoint types, see:
 
 See information on specific methods of obtaining and setting the access keys within the provider-specific documentation within this directory.
 
-## Step 4: Starting the MLflow Deployments server
+## Step 4: Starting the MLflow AI Gateway
 
-With the MLflow configuration file in place and access key(s) set, you can now start the MLflow Deployments server.
+With the MLflow configuration file in place and access key(s) set, you can now start the MLflow AI Gateway.
 Replace `<provider>` with the actual path to the MLflow configuration file for the provider of your choice:
 
 ```sh
-mlflow deployments start-server --config-path examples/deployments/deployments_server/<provider>/config.yaml --port 7000
+mlflow gateway start --config-path examples/deployments/deployments_server/<provider>/config.yaml --port 7000
```

Comment:
Shouldn't we change config paths in these examples?

Reply:
thanks for the catch, yes!